xarray/tests/test_backends.py

```diff
@@ -1673,6 +1681,10 @@ def test_orthogonal_indexing(self):
         # pynio also does not support list-like indexing
         pass
 
+    def test_orthogonal_isel(self):
+        # pynio also does not support list-like indexing
```

Comment:
do we raise `NotImplementedError` with pynio when anything other than `BasicIndexer` is used?